src/components/bookDetails/Image.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+const BookImage = {
+  width: "412px",
+  height: "653px",
+  borderRadius: "10px",
+};
+
+const Image = ({imgApi}) => {
+  return <img src={imgApi} alt="hi" style={BookImage} />;

```

Comment:
class name should be camelCase not BookImage but bookImageStyle for example
 since BookImage you will think automatically it is Component